src/errors.js

```diff
@@ -52,5 +52,7 @@ export const ErrorMessages = {
   STRICT_LHS_POSTFIX: "Postfix increment/decrement may not have eval or arguments operand in strict mode",
   STRICT_LHS_PREFIX: "Prefix increment/decrement may not have eval or arguments operand in strict mode",
   STRICT_RESERVED_WORD: "Use of future reserved word in strict mode",
-  ILLEGAL_ARROW_FUNCTION_PARAMS: "Illegal arrow function parameter list"
+  ILLEGAL_ARROW_FUNCTION_PARAMS: "Illegal arrow function parameter list",
+  INVALID_VAR_INIT_FI: "Invalid variable declaration in for-in statement",
```

Comment:
Shall we use `...FOR_IN`? I think we should not encourage use of acronyms... The only one we used is LHS for LeftHandSideExpression.